products/sapmachine.md

```diff
@@ -0,0 +1,98 @@
+---
+title: SapMachine
+category: lang
+tags: sap java-distribution
+iconSlug: openjdk
```

Comment:
```suggestion
iconSlug: sap
```

Reply:
Similar to https://github.com/endoflife-date/endoflife.date/pull/3191#discussion_r1241853145, I think the OpenJDK icon makes more sense in this context.